### ISSUE TITLE
CompactInline and Datefield

### ISSUE THREAD

**Issue by rdollinger** [geex-arts/django-jet#179]
Hi,

I have a problem with an Datefield and the CompactInline. When I add a new record in the CompactInline and I click on the calendar symbol, the calendar doesn't appear and instead the main form is displayed. Here is the error I found in the console:

`
bundle.min.js?v=1.0.4:3 Uncaught TypeError: Cannot read property 'left' of undefined
    at i._findPos (bundle.min.js?v=1.0.4:3)
    at i._showDatepicker (bundle.min.js?v=1.0.4:3)
    at HTMLInputElement.<anonymous> (bundle.min.js?v=1.0.4:4)
    at Function.each (bundle.min.js?v=1.0.4:7)
    at ot.fn.init.each (bundle.min.js?v=1.0.4:7)
    at ot.fn.init.t.fn.datepicker (bundle.min.js?v=1.0.4:4)
    at HTMLAnchorElement.<anonymous> (bundle.min.js?v=1.0.4:1)
    at HTMLAnchorElement.dispatch (bundle.min.js?v=1.0.4:9)
    at HTMLAnchorElement.m.handle (bundle.min.js?v=1.0.4:8)
_findPos @ bundle.min.js?v=1.0.4:3
_showDatepicker @ bundle.min.js?v=1.0.4:3
(anonymous) @ bundle.min.js?v=1.0.4:4
each @ bundle.min.js?v=1.0.4:7
each @ bundle.min.js?v=1.0.4:7
t.fn.datepicker @ bundle.min.js?v=1.0.4:4
(anonymous) @ bundle.min.js?v=1.0.4:1
dispatch @ bundle.min.js?v=1.0.4:9
m.handle @ bundle.min.js?v=1.0.4:8
`

When I click in the input field instead of the calendar symbol everything works perfectly. However when I change an existing record clicking on the input field and on the calendar symbol everything works perfectly.

**Comment by Ovenwand** [comment]
I'm experiencing the same problem, this also applies to both Datefield and Timefield. When a new row is added, the datepicker/timepicker events aren't bound to the input or the button.

**Comment by SalahAdDin** [comment]
@f1nality other error with `jquery`?

**Comment by Ismael-VC** [comment]
Please come to the django-jet Discord server so we can organize if you like:

  *   https://discord.gg/3yr5yeH

Welcome! 😄

**Comment by SalahAdDin** [comment]
@rdollinger Do you still have this problem?

**Comment by rdollinger** [comment]
Yes I have still this problem. I'm using django-jet==1.0.6

**Comment by SalahAdDin** [comment]
Yes, please, a gift will be good, we need a live example for debug it.

**Comment by rdollinger** [comment]
Sorry for the late answer.

I have made a little example project. You can find it here:

https://gitlab.com/rdollinger/jet-test <https://gitlab.com/rdollinger/jet-test>

You can simulate it by clicking on „Auftragspositionen“ and then create or update an existing record. In the „auftragspoisitonen-record" you now add a new record in the CompactInline and click on the calendar symbol, the calendar doesn't appear and instead the main form (auftragpositionen-record) is displayed.  When I click in the input field instead of the calendar symbol everything works perfectly. However when I change an existing "leistungen-record clicking on the input field and on the calendar symbol everything works perfectly.

I hope the example can help you

bye
Robert

**Comment by SalahAdDin** [comment]
@rdollinger Did you solve it?

**Comment by rdollinger** [comment]
Sorry no. I have made an little example where you can simulate it

https://gitlab.com/rdollinger/jet-test

Bye
Robert

**Comment by SalahAdDin** [comment]
@Ismael-VC What do you think about this?

**Comment by abdulwahid24** [comment]
@Ismael-VC @SalahAdDin Do you have any update on it?

**Comment by SalahAdDin** [comment]
Not now, whe are waiting for PR.
